### ISSUE TITLE
Updated from 1.0.9 to 1.0.11 and it broke [object Object] has no method '_setLookupHost'

### ISSUE THREAD

**Issue by bogdanrn** [iron-meteor/iron-router#1452]
This is the error i'm getting now 
```
TypeError: Object [object Object] has no method '_setLookupHost'
    at [object Object].Controller (packages/iron_controller/lib/controller.js:25:1)
    at new Controller.extend.constructor (packages/iron_router/lib/route_controller.js:14:1)
    at [object Object].ctor (packages/iron_core/lib/iron_core.js:88:1)
    at Function.Router.createController (packages/iron_router/lib/router.js:201:1)
    at Function.Router.dispatch (packages/iron_router/lib/router_server.js:39:1)
    at Object.router (packages/iron_router/lib/router.js:15:1)
    at next ```

**Comment by chrisbutler** [comment]
it looks like there might be an issue with a subproject version. i would suggest downgrading to 1.0.10 but it looks like meteor's constraint solver isn't working as expected and just grabs the latest versions of the subprojects... can you try and see if you have any luck?

**Comment by bogdanrn** [comment]
I already tried to revert back to 1.0.9 but the app is still broken. Can you point me on how to downgrade subproject ?

**Comment by pfafman** [comment]
Getting the same here

**Comment by floodpeak** [comment]
meteor add iron:router@=1.0.10
meteor add iron:core@=1.0.8

**Comment by bogdanrn** [comment]
@floodpeak  Thanks, this makes it work for now.

**Comment by zachariahtimothy** [comment]
Confirmed @floodpeak 's solution works. I was downgrading like crazy but did not downgrade the core. Thanks!

**Comment by chrisbutler** [comment]
@bogdanrn @pfafman @floodpeak @zachariahtimothy should be fixed now in 1.0.12
